src/Compilers/Server/VBCSCompilerTests/DesktopBuildClientTests.cs

```diff
@@ -154,8 +155,7 @@ public void TestMutexConstructorException()
                 {
                     Assert.True(createdNew);
                     var mutexSecurity = outer.GetAccessControl();
-                    var user = Environment.UserDomainName + "\\" + Environment.UserName;
-                    mutexSecurity.AddAccessRule(new MutexAccessRule(user, MutexRights.FullControl, AccessControlType.Deny));
+                    mutexSecurity.AddAccessRule(new MutexAccessRule(WindowsIdentity.GetCurrent().Owner, MutexRights.FullControl, AccessControlType.Deny));
```

Comment:
Did this slip in by mistake?

Reply:
Nope. The original code doesn't behave as expected in the VSTS runs. This is a deliberate change to try and get the test passing here. See swat channel.